storage/types.go

```diff
@@ -93,7 +93,8 @@ func GenerateRandomChunk(dataSize int64) Chunk {
 	sdata := make([]byte, dataSize+8)
 	rand.Read(sdata[8:])
 	binary.LittleEndian.PutUint64(sdata[:8], uint64(dataSize))
-	hasher.ResetWithLength(sdata[:8])
+	hasher.Reset()
```

Comment:
now this is called twice

Reply:
I'm sorry I don't understand what you mean

Reply:
yes